R/smoothAndRecombine.R

```diff
@@ -78,36 +73,18 @@ setMethod("smoothAndRecombine",
           function(gene_expression, adj_matrix, alpha,
                    smoothing.function=randomWalkByMatrixInv,
                    normalizeAdjMatrix=c('rows','columns'),
-                   chunk.size = 1,
                    filepath = NULL) {
-            normalizeAdjMatrix <- match.arg(normalizeAdjMatrix)
+            #normalizeAdjMatrix <- match.arg(normalizeAdjMatrix)
```

Comment:
Is there a reason this is commented and `'columns'` is hard-coded 5 rows later down?